test/unit/creation_test.rb

```diff
@@ -18,35 +18,35 @@ def after_create_method
   class Author < TestResource
   end
 
-  def setup
-    super
+  def stub_simple_creation
```

Comment:
when this code were in setup in cause falls positive test `#test_create_with_relationships_in_payload`
because in this test create request has different payload (and different stub) but it pass because stub from `#setup` aws matched instead of stub written in test itself.

I move stub into separate method and use it only in tests that relies on it. it will save us from future falls positive tests.